clcache/monkey.py

```diff
@@ -0,0 +1,59 @@
+import os
+import sys
+import shutil
+
+from typing import List
+from contextlib import suppress
+from os.path import join, dirname, isfile
+
+def patch_distutils() -> None:
+    # Try to import numpy.distutils first so that we
+    # can patch after numpy does (We're very early in
+    # the import cycle)
+
+    with suppress(ImportError):
+        from numpy.distutils import ccompiler
+
+    from distutils import ccompiler
+    
+    # Make some reasonable assumptions about the Python directory structure
+    clcache_path = None
+    if True:
+        clcache_path = join(dirname(sys.executable), 'Scripts', 'clcache.exe')
```

Comment:
I think there's no reason anymore to keep lines 20 and 21.

Reply:
It looks nice, but will revert.